mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Surgex.Mixfile do
   def project do
     [
       app: :surgex,
-      version: "4.2.0",
+      version: "4.3.0",
```

Comment:
I think this is either a patch (because we're fixing an incomplete implementation), or a breaking change (because we will suddenly return values we didn't before. I'd go for a patch version update as I wouldn't expect anyone to rely on a `FunctionClauseError` being raised. WDYT?

Reply:
Ok, makes sense to make it a patch update.